src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/CaseService.java

```diff
@@ -13,13 +15,23 @@
 public interface CaseService {
 
   /**
-   * Retrieve the data relating to a Case
+   * Retrieve the data relating to HH Cases by address UPRN
    *
-   * @param uprn of a Case
-   * @return List of Case details for a UPRN
+   * @param uprn of address for which HH case details are requested
+   * @return List of HH Case details for address UPRN
    * @throws CTPException
    */
   List<CaseDTO> getHHCaseByUPRN(final UniquePropertyReferenceNumber uprn) throws CTPException;
 
-  public void fulfilmentRequestBySMS(SMSFulfilmentRequestDTO requestBodyDTO) throws CTPException;
+  /**
+   * Change address for a Case
+   *
+   * @param caseId UUID for case to which address relates
+   * @param AddressChangeDTO for change of address
+   * @return Case details for changed address
+   * @throws CTPException
+   */
+  CaseDTO modifyAddress(final UUID caseId, final AddressChangeDTO address) throws CTPException;
```

Comment:
The caseId is also in the address change DTO - can remove the first param here

Reply:
Removed param, checked request body matches path variable caseId in endpoint where service is called.